man/rhsmcertd.8

```diff
@@ -93,22 +87,22 @@ service rhsmcertd stop
 rhsmcertd --cert-check-interval=240
 .fi
 
-.SS RUNNING CERTIFICATE AND HEALING SCANS IMMEDIATELY
-Normally, the certificate and auto-attach scans are run periodically, on a schedule defined in the \fBrhsmcertd\fP configuration. The scans can be run immediately -- which is useful if an administrator knows that there are new subscriptions available -- and then the scans resume their schedules.
+.SS RUNNING CERTIFICATE SCANS IMMEDIATELY
+Normally, the certificate scans are run periodically, on a schedule defined in the \fBrhsmcertd\fP configuration. The scans can be run immediately -- which is useful if an administrator knows that there are new subscriptions available -- and then the scans resume their schedules.
 .nf
 service rhsmcertd stop
 rhsmcertd -n
 .fi
 
 .SS DEPRECATED USAGE
-\fBrhsmcertd\fP used to allow the certificate and auto-attach intervals to be reset simply by passing two integers as arguments.
+\fBrhsmcertd\fP used to allow the certificate intervals to be reset simply by passing an integer as arguments.
```

Comment:
nit: "as argument" (only one)

Reply:
Thank you @ptoscano, good catch. 
Address by commit [2e6f959](https://github.com/candlepin/subscription-manager/pull/3455/commits/2e6f959935887e122c70a56e156fb04e9c33a6aa)